_topic_map.yml

```diff
@@ -496,6 +494,8 @@ Topics:
   - Name: Securing Builds by Strategy
     File: securing_builds
     Distros: openshift-origin,openshift-enterprise
+  - Name: Seccomp
```

Comment:
Is this a sufficient title? Should you use the long form of "Secure Computing Mode"?